package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mobietrain/eslint-config",
-  "version": "2.0.0",
+  "version": "3.0.0",
```

Comment:
This change is going to be applied automatically as long as you follow [those guidelines](https://github.com/googleapis/release-please#how-should-i-write-my-commits).

Reply:
Fixed.